tests/integration/tier0/check-custom-repo/test_custom_repos.py

```diff
@@ -5,8 +5,13 @@
 import pytest
 
 
-def get_system_version(system_release_content=None):
-    """Return a namedtuple with major and minor elements, both of an int type.
+# TODO move to conftest
```

Comment:
Can we get a handler here? 

```suggestion
# TODO(danmyway) move to conftest
```

Reply:
Isn't it moved to the the conftest file in this PR already?

Reply:
It is there in a somewhat different form. I will keep the cleanup for a different PR.